test/test_grudge.py

```diff
@@ -1109,47 +1096,77 @@ def test_norm_obj_array(actx_factory, p):
     # }}}
 
 
-def test_map_if(actx_factory):
-    """Test :meth:`grudge.symbolic.execution.ExecutionMapper.map_if` handling
-    of scalar conditions.
-    """
+def test_empty_boundary(actx_factory):
+    # https://github.com/inducer/grudge/issues/54
+
+    from meshmode.mesh import BTAG_NONE
 
     actx = actx_factory()
 
     dim = 2
     mesh = mgen.generate_regular_rect_mesh(
             a=(-0.5,)*dim, b=(0.5,)*dim,
             nelements_per_axis=(8,)*dim, order=4)
-    discr = DiscretizationCollection(actx, mesh, order=4)
+    dcoll = DiscretizationCollection(actx, mesh, order=4)
+    normal = op.normal(dcoll, BTAG_NONE)
+    from meshmode.dof_array import DOFArray
+    for component in normal:
+        assert isinstance(component, DOFArray)
+        assert len(component) == len(dcoll.discr_from_dd(BTAG_NONE).groups)
 
-    sym_if = sym.If(sym.Comparison(2.0, "<", 1.0e-14), 1.0, 2.0)
-    bind(discr, sym_if)(actx)
 
+# {{{ Deprecated symbolic tests
```

Comment:
Symbolic specific tests which do not have a "new world grudge" equivalent are moved down here